graphsemnet/semantics.py

```diff
@@ -35,3 +37,37 @@ def semantic_dsm_safe(word_list, keyed_vectors):
     vectors = np.array(vectors)
     matrix = pdist(vectors, metric='cosine')
     return (matrix, labels)
+
+
+class SemanticGraph(object):
+    """A graph where nodes are concepts and edges are association strengths.
+    """
+    def __init__(self, adj=None, labels=None, dsm=None, directed=True):
```

Comment:
Do we really need to have both adj and dsm? I think the logic could be moved outside, and follow the standard definition for a graph with adjacency matrix and labels. So `adj` would be a required argument, and the rest kwargs

Reply:
maybe, but let's do that later—there are a lot of details that need changing, but are outside the scope of the goal of this pull request